unittests/wasm_config_tests.cpp

```diff
@@ -418,6 +418,75 @@ BOOST_DATA_TEST_CASE_F(old_wasm_tester, max_func_local_bytes_old, data::make({0,
    }
 }
 
+// Combines max_call_depth and max_func_local_bytes
+BOOST_FIXTURE_TEST_CASE(max_stack, wasm_config_tester) {
```

Comment:
The test name seems kind of a misnomer --  is it really testing the maximum? Or just something kinda large. My impression is that it's just kinda large.

Reply:
You're right.  It's one frame short of the max, since I didn't increase the stack usage of `apply`

Reply:
If we're going to test the max it'd be nice to see a max+1 test fail too.

Reply:
I don't think that a combined max+1 test adds anything over the separate max+1 tests for max_call_depth and max_func_local_bytes which already exist.

Reply:
yeah, gotcha. I believe I was thinking of a different test